server/src/main/java/com/ncc/neon/better/IENlpModule.java

```diff
@@ -16,11 +19,13 @@
 public class IENlpModule extends NlpModule {
     private final String TRAIN_OUTPUTS_KEY = "train_outputs";
     private final String INF_OUTPUTS_KEY = "inf_outputs";
+    private final String JOB_ID_KEY = "job_id";
```

Comment:
Should this be identical to the JOB_ID_KEY defined before? If so, it would be better to use the same identifier for both instead of repeating the string.